wgpu-core/src/command/bundle.rs

```diff
@@ -104,6 +104,7 @@ pub struct RenderBundle {
 }
 
 unsafe impl Send for RenderBundle {}
+
```

Comment:
hmm, another rustfmt thing? We are running it locally all the time, so I'm surprised you see changes.
My guess is you are using another (some weird?) version of rustfmt. We use the stable one that comes with `rustup comoponent add rustfmt`. Please double-check that, there shouldn't be formatting changes outside of your code - it's all already formatted.